jina/types/arrays/search_ops.py

```diff
@@ -0,0 +1,68 @@
+import re
+import operator
+from typing import Dict
+
+
+if False:
+    from .document import DocumentArray
+
+
+class DocumentArraySearchOpsMixin:
+    """ A mixin that provides search functionality to DocumentArrays"""
+
+    operators = {
+        '<': operator.lt,
+        '>': operator.gt,
+        '==': operator.eq,
+        '!=': operator.ne,
+        '<=': operator.le,
+        '>=': operator.ge,
+    }
+
+    def find(
+        self,
+        regexes: Dict,
+        traversal_paths: list = ['r'],
+        operator: str = '>=',
+        value: int = 1,
```

Comment:
```suggestion
        value: Optional[int] = None,
```

And then in the code say.

```python
value = value or len(regexes)
```

Reply:
yes, done